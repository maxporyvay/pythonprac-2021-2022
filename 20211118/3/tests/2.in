try:
	alp = Alpha(c=10, z=2, ab=42)
except AttributeError:
	print('catched')
alq = AlphaQ(c=10, z=2, a=42)
try:
	alq.E = 123
except AttributeError:
	print('catched')
print(alq)