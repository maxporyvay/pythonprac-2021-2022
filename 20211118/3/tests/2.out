catched
catched
a: 42, c: 10, z: 2
